packages/components/package.json

```diff
@@ -17,11 +17,11 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/schwarzit/nui",
-    "directory": "packages/nui"
+    "url": "https://github.com/schwarzit/onyx",
+    "directory": "packages/components"
```

Comment:
Should we also rename the folder from `components` to `sit-onyx`?
So that the folder name is equal to the package name for every package

Reply:
done